addons/t9n/static/src/core/copy_button_popover.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<templates>
+    <t t-name="t9n.copyButtonPopover">
+        <div class="popover-body">Copied to clipboard!</div>
+    </t>
+</templates>
```

Comment:
Using a popover was really not a bad idea, but for this kind of stuff we'd rather use a "notification" (look at the code for notificationService)